pkg/helm3/schema/schema.json

```diff
@@ -62,6 +62,10 @@
                 "type":"string"
               }
             },
+            "atomic": {
+              "type":"boolean",
+              "default": true
```

Comment:
Here's a bit of background on how default works in the json schema, it mostly controls how VS Code does autocomplete when people fill out porter.yaml

When default is true, what will happen is that if they accept the autocomplete from VS Code, it will add all of the defaulted properties on the helm3 mixin into the porter.yaml. So everyone will get this:

```yaml
install:
- helm3:
     atomic: true
```

If we want to match the current behavior and have people leave atomic unspecified by default (and letting the mixin default to true), then the default property in the json schema should be omitted.

What do you think?

Reply:
Thanks for the background - I must admit I just looked at the other properties and assumed that `default` was the, well, default value for the property 😉 

With the additional knowledge I agree with you that `default` property should be omitted. I'll fix it right away (and the other stuff as well). 👍